modules/installation/pages/client-requirements.adoc

```diff
@@ -6,20 +6,26 @@
 
 Supported operating systems for traditional and Salt clients are listed in this table.
 
+In this table, &#2713; indicates that clients running the operating system are supported by {scc}, and &#2717; indicates that it is not supported.
+Fields marked as `TBD` are under consideration, and may or may not be supported at a later date.
+
 [[mgr.supported.clients]]
 [cols="1,1,1,1", options="header"]
 .Supported Client Systems
 |===
-| Operating Systems                                    | Architecture                                 | Traditional Clients | Salt Clients
-| {sle} 11 SP4                                         | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported           | Supported
-| {sle} 12 SP3, 12 SP4                                 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported         | Supported
-| {sle} 15                                             | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported         | Supported
-|_Latest minor release {rhnminrelease6}_               | {x86}, {x86}_64                              | Supported           | Supported
-|_Latest minor release {rhnminrelease7}_               | {x86}_64                                     | Supported           | Supported
-| Open Enterprise Server 2015, 2015 SP1, 2018          | {x86}_64                                     | Supported           | Supported
-| {opensuse} Leap 15.1                                 | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 16.04                                       | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 18.04                                       | {x86}_64                                     | Unsupported         | Supported
+| Operating System | Architecture | Traditional Clients | Salt Clients
+| {sle} 15 | {x86}_64, {ppc}, {ibmz}, ARM | &#2713; | &#2713;
+| {sle} 12 | {x86}_64, {ppc}, {ibmz}, ARM | &#2713; | &#2713;
+| {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz} | &#2713; | &#2713;
+| {rhel} 8 | {x86}_64 | TBD | TBD
+| {rhel} 7 | {x86}_64 | &#2713; | &#2713;
+| {rhel} 6 | {x86}, {x86}_64 | &#2713; | &#2713;
```

Comment:
Do we need to name "Expanded Support" extra? 

SLES-ES 6 / 7 is supported like RHEL.

Reply:
Added.